<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CanViewOptionDialog</class>
 <widget class="QDialog" name="CanViewOptionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>387</width>
    <height>171</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>设置</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QGridLayout" name="gridLayout">
      <item row="2" column="0">
       <widget class="QtMaterialCheckBox" name="chkDirection">
        <property name="text">
         <string>方向</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QtMaterialCheckBox" name="chkFrameType">
        <property name="text">
         <string>帧类型</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QtMaterialCheckBox" name="chkFormat">
        <property name="text">
         <string>帧格式</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QtMaterialCheckBox" name="chkCanType">
        <property name="text">
         <string>CAN类型</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QtMaterialCheckBox" name="chkId">
        <property name="text">
         <string>帧ID</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QtMaterialCheckBox" name="chkData">
        <property name="text">
         <string>数据</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QtMaterialCheckBox" name="chkLength">
        <property name="text">
         <string>长度</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QtMaterialCheckBox" name="chkStamp">
        <property name="text">
         <string>时间标识</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>列表显示项:</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QtMaterialFlatButton" name="btnOk">
        <property name="text">
         <string>确定</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QtMaterialCheckBox</class>
   <extends>QCheckBox</extends>
   <header>qtmaterialcheckbox.h</header>
  </customwidget>
  <customwidget>
   <class>QtMaterialFlatButton</class>
   <extends>QPushButton</extends>
   <header>qtmaterialflatbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
